src/Sarif.ValidationTests/SarifValidatorTests.cs

```diff
@@ -113,6 +114,17 @@ private void ValidateDefaultDocument(IDictionary<Type, DefaultObjectPopulatingVi
             SarifLog clonedLog = sarifLog.DeepClone();
             clonedLog.ValueEquals(sarifLog).Should().BeTrue();
 
+            JObject jObjectSarifLog = JObject.Parse(toValidate);
+
+            // The following validations ensure NotYetAutoGenerated fields work as expected.
```

Comment:
>// The following validations ensure NotYetAutoGenerated fields work as expected. [](start = 12, length = 80)

@mikefan i tried two things here for "Run.ColumnKind" validation:

1. use local SarifLog object for the validation:
        newSarifLog.Runs[0].ColumnKind.Should().Be(ColumnKind.Utf16CodeUnits);
 however, this fails because our logic populates the default value, which according to our schema is ColumnKind.UnicodeCodePoints.

2. i added code to initialize a new run object (like you suggested in the first place, but it took me while to understand why #1 wont work :)). However i changed my mind about it and reverted it because of two reasons:
      a. this code is run for a plethora of files, it felt weird adding that test here which actually has no relation to the current file.
      b. removal of not-yet-autogen/run file will end up failing many converter test cases so this will be hard to miss.

Perhaps there s a simple UTC we should add instead (like you suggested at one point) and now i agree :).